Maestro/subscriptions.json

```diff
@@ -530,6 +530,32 @@
         }
       }
     },
+    // Update dependencies in core-setup release/2.1
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/release/2.1/Latest.txt"
```

Comment:
Any reason not to add corefx and coreclr triggers here?

/cc @weshaggard

Reply:
Doesn't hurt anything to trigger on paths that don't exist, but it could be confusing since Core-Setup dependencies.props can only take BuildTools upgrades. (Because that will 404 if it's configured to upgrade on a build-info that doesn't exist, and updaters would be confused about packages that don't exist, too.)

Also not sure yet if we're going to uptake prodcon 2.1 builds through the individual build-infos: https://github.com/dotnet/core-eng/issues/2435. Doesn't necessarily impact what we do now, though.